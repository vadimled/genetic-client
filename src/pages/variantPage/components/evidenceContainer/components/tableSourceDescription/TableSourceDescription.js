import React from "react";
import PropTypes from "prop-types";
import style from "./TableSourceDescription.module.scss";

function TableSourceDescription({ source, description }) {
  return (
    <div className={style["table-source-description-wrapper"]}>
      <div className="table-source-description">
        <div className="source">{source}</div>
        <div className="description">{description}</div>
      </div>
    </div>
  );
}

TableSourceDescription.propTypes = {
  source: PropTypes.string,
  description: PropTypes.string
};

export default TableSourceDescription;
