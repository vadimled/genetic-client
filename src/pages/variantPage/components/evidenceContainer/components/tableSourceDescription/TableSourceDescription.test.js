import React from 'react';
import { shallow } from 'enzyme';
import TableSourceDescription from './TableSourceDescription';

describe('<TableSourceDescription />', () => {
  test('renders', () => {
    const wrapper = shallow(<TableSourceDescription />);
    expect(wrapper).toMatchSnapshot();
  });
});
