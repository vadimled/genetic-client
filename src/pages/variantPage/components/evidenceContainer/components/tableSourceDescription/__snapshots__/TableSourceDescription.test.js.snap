// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableSourceDescription /> snapshot 1`] = `
<DocumentFragment>
  <div
    class="table-source-description-wrapper"
  >
    <div
      class="table-source-description"
    >
      <div
        class="source"
      />
      <div
        class="description"
      />
    </div>
  </div>
</DocumentFragment>
`;
